src/learning/gmm.rs

```diff
@@ -30,75 +29,82 @@
 //! // Probabilities that each point comes from each Gaussian.
 //! println!("{:?}", post_probs.data());
 //! ```
-use linalg::{Matrix, MatrixSlice, Vector, BaseMatrix, BaseMatrixMut, Axes};
-use rulinalg::utils;
+extern crate rand;
```

Comment:
Shouldn't need this as we import the crate in `lib.rs`. Just `use rand` should work.